Makefile

```diff
@@ -92,5 +92,14 @@ generate-api:
 vps:
 	VBoxManage list runningvms
 
+tt:
```

Comment:
I'm happy with `make reload`. You can always have an alias for you locally if you like `make tt`